docs/onboarding/Onboarding.md

```diff
@@ -0,0 +1,125 @@
+
+The endpoint accepts the following parameters:
+* `recipient`
+  * An uuid used by [Cloud Connector](https://github.com/RedHatInsights/cloud-connector) to route the playbook signal. This field is required.
+    * For `rhc-worker-playbook` or "directly connected hosts", `recipient` can be obtained from the [system profile](https://github.com/RedHatInsights/inventory-schemas/blob/8000191d960da05c4ebf7960f4af8f7cf68bf616/schemas/system_profile/v1.yaml#L197).
+    * For satellite hosts, this will be provided by the [Sources Api](https://github.com/RedHatInsights/sources-api-go), where it is know as the `rhc_id`.
+    The `rhc_id`, or `recipient`, can be obtained using the `/sources/{id}/rhc_connections` [endpoint](https://github.com/RedHatInsights/sources-api-go/blob/9a7c9288be53f84717a6337063a481dcf533f615/public/openapi-3-v3.1.json#L1333).
+    For example:
+      ```sh
+        curl -v -H "x-rh-identity: <identity-header>" localhost:3000/api/sources/v3.1/sources/<source-id>/rhc_connections      
+      ```
+* `org_id`
+  * The account/tenant this playbook is part of.
+    This field is required.
+* `principal`
+  * Username of the user this playbook belongs to.
+    This field is required.
+* `url`
+  * The url that is hosting the playbook.
+    This url will be used to execute the playbook.
+    This field is required.
+    Also, a playbook that needs to be run by `rhc-worker-playbook`, must and only point to the `localhost` information under the `hosts` information.
+* `name`
+  * The name of the playbook.
+    Human readable name of the playbook run.
+    Used to present the given playbook run in external systems (Satellite).
+    This field is required.
+* `web_console_url`
+  * The url where the user can find more information on the playbook run.
+    Optional, but highly suggested.
+* `labels`
+  * Metadata for the playbook run.
+    This will be helpful when filtering playbook runs.
+* `timeout`
+  * Number of seconds to wait before the playbook run is marked as failed due to timeout.
+* `hosts`
+  * An array of objects containing information on the hosts involved in the run.
+    Please refer to [this section](https://github.com/RedHatInsights/playbook-dispatcher/blob/master/schema/private.openapi.yaml#L202) of the schema for more infomation.
+    For satellite playbook requests, the `inventory_id` will need to be provided for each hosts.
+* recipient_config
+  * Recipient config for satellite.
+    Required for satellite playbooks.
+    * `sat_id` - Satellite instance id.
+      Can be obtained from [host records provided by Inventory](https://github.com/RedHatInsights/insights-host-inventory/blob/9d2c837ee37a6fe50b628880ac5d823319749a82/swagger/api.spec.yaml#L901).
+    * `sat_org_id` - Satellite organization id.
+      This is present within [Inventory's host Facts](https://github.com/RedHatInsights/insights-host-inventory/blob/4e09b9154c364d2553c259cfeef2b99772aef06d/swagger/api.spec.yaml#L848).
+
```

Comment:
One piece of information that is missing is that there should be a separate request for each directly connected host whereas hosts connected via satellite should be grouped together under one sat request (using the hosts property)